.github/workflows/push-csv-to-repo.yml

```diff
@@ -3,43 +3,50 @@ name: Push CSV to Repo
 
 on:
   workflow_dispatch: # manual trigger 
-  push:
-    branches:
-      - prannoy/workflow-setup
-  # schedule:
-  #   - cron: '0 12 * * *' # runs at 12:00 UTC everyday
+  schedule:
+    - cron: '0 12 * * *' # runs at 12:00 UTC everyday
 
 jobs:
   build:
     runs-on: ubuntu-latest
-    container: pandoc/latex
     steps:
-      - name: Checkout repo
+      - name: Checkout iit-backend
         uses: actions/checkout@v2
-      - name: Setup Python
+        with:
+          path: iit-backend
+
+      - name: Setup python
         uses: actions/setup-python@v2
         with:
-          python-version: '3.8.x'
-      - name: Install python packages
-        run: |
-          pip install -r requirements.txt
-      # - name: Execute py script # run estimate_csv_creator.py to get CSV
-      #   run: python3 app/github_public_repo/estimate_csv_creator.py
-      # - name: Copy CSV
-      #   uses: andstor/copycat-action@v3
-      #   with:
-      #     personal_token: ${{ secrets.API_TOKEN_GITHUB }}
-      #     src_path: /output/.
-      #     dst_path: /
-      #     exclude: "*.py"
-      #     dst_owner: serotracker
-      #     dst_repo_name: sars-cov-2-data
-      #     dst_branch: test
-      #     src_branch: simona/github_csv_driver_script
-      #     src_wiki: false
-      #     dst_wiki: false
-      #     username: serosurveillance-can
-      #     email: can.serosurveillance.dev@gmail.com
-     
+          python-version: '3.8.3' # Version range or exact version of a Python version to use, using SemVer's version range syntax
+          architecture: 'x64' # optional x64 or x86. Defaults to x64 if not specified
+
+      - name: Install packages
+        uses: BSFishy/pip-action@v1
+        with:
+          requirements: iit-backend/requirements.txt
+  
+      - name: Create CSV locally  # run estimate_csv_creator.py to get CSV
+        run: python3 $GITHUB_WORKSPACE/iit-backend/app/github_public_repo/estimate_csv_creator.py
+        env:
+          AIRTABLE_API_KEY: ${{ secrets.AIRTABLE_API_KEY }}
+          AIRTABLE_BASE_ID: ${{ secrets.AIRTABLE_BASE_ID }}
+          FLASK_ENV: 'dev'
+          LOG_CONFIG_PATH: 'iit-backend/logging.cfg'
+          LOG_FILE_PATH: 'iit-backend/logfile.log'
+          PYTHONPATH: 'iit-backend'
+    
+      - name: Copies CSV to sars-cov-2-data
```

Comment:
This step just copies the CSV to the iit backend repo -- where is the step that actually pushes this to the main branch? Or does this do this automatically on the main branch?

Reply:
Based on the action docs (https://github.com/marketplace/actions/push-a-file-to-another-repository), seems like it actually does push the file as well.

@annoyingpr can you update the step name to be clear that both copying and pushing are happening here? Thanks

Reply:
yeah just realized that the pushing part wasn't as clear, will update the step name!